readme.md

```diff
@@ -94,7 +111,8 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+OAuth is using a token verified thru the token provider such as google, or facebook.
+Which, in case of a valet, you are borrowing the key to park the car.
```

Comment:
Yeah! Also, specifically valet keys refer to keys which will start an ignition but not open the trunk/glovebox. It is permission to use a restricted set of features of the service (3rd party authenticator/car).